scripts/export_esm.py

```diff
@@ -0,0 +1,49 @@
+import json
+import pandas as pd
+import csv
+import sys
+
+def main(pid,week):
+    if week == "1":
+        has_intervention = False
+    elif week == "2":
+        has_intervention = True
+    else:
+        print("Unknown week parameter: ",week)
+        return 
+    
+    # read file
+    df = pd.read_json('../../purpose-mode-data/'+pid+'/esm.json', lines=True)
```

Comment:
I'd replace the hard-coded relative paths with a command-line argument but no big deal; we can leave this for now.